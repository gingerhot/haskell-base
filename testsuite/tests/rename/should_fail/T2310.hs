{-# OPTIONS_GHC -XScopedTypeVariables #-}

module Foo where

foo = let c = \ (x :: a) -> (x :: a) in co
