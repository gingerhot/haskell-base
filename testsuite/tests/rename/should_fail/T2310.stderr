
T2310.hs:5:41: error:
    • Variable not in scope: co
    • Perhaps you meant one of these:
        ‘c’ (line 5), ‘cos’ (imported from Prelude)
